frontend/packages/operator-lifecycle-manager/src/utils/filters.ts

```diff
@@ -0,0 +1 @@
+export * from '@console/operator-lifecycle-manager-v1/src/utils/filters';
```

Comment:
can we make sure that only olmv0 code is importing from this path, so it's easier to remove the olmv0 static plugin in the future?